src/commands/social/rep.js

```diff
@@ -10,6 +10,8 @@ module.exports = class Rep extends Command {
     this.name = 'rep'
     this.category = 'social'
 
+    this.requirements = new CommandRequirements(this, { onlyOldAccounts: true, databaseOnly: true })

```

Comment:
I love how this didn't have databaseOnly already

Reply:
HEUHEUEHUEHEUHEU